MenMa WatchKit Extension/InterfaceController.swift

```diff
@@ -58,11 +58,13 @@ class InterfaceController: WKInterfaceController, WCSessionDelegate {
         
         let messageToSend = ["value": "message sent successfully"]
         session.sendMessage(messageToSend, replyHandler: { replyMessage in
-            //            //handle and present the message on screen
-            //            let value = replyMessage["value"] as? Venue
-            //            //            self.messageLabel.setText(value)
+            //handle and present the message on screen
+            let value = replyMessage["value"] as? Venue
```

Comment:
At line 85 in MainViewController for the iOS, replyHandler call is commented out, so this line replyMessage should be empty?

In MainViewController, session.sendMessageData sends encoded venues.

With the watch, InterfaceController is received venues data in session:didReceiveMessageData:replyHandler: method.